api/grpc/disperser/v2/disperser_v2.pb.go

```diff
@@ -25,26 +25,31 @@ const (
 // BlobStatus represents the status of a blob.
 // The status of a blob is updated as the blob is processed by the disperser.
 // The status of a blob can be queried by the client using the GetBlobStatus API.
-// Intermediate states are states that the blob can be in while being processed, and it can be updated to a differet state:
+// Intermediate states are states that the blob can be in while being processed, and it can be updated to a different state:
 // - QUEUED
 // - ENCODED
 // Terminal states are states that will not be updated to a different state:
 // - CERTIFIED
 // - FAILED
 // - INSUFFICIENT_SIGNATURES
+// - UNKNOWN
 type BlobStatus int32
 
 const (
+	// UNKNOWN means that the status of the blob is unknown.
+	// This is a catch all and should not be encountered absent a bug.
 	BlobStatus_UNKNOWN BlobStatus = 0
-	// QUEUED means that the blob has been queued by the disperser for processing
+	// QUEUED means that the blob has been queued by the disperser for processing.
+	// The DisperseBlob API is asynchronous, meaning that after request validation, but before any processing,
+	// the blob is stored in a queue of some sort, and a response immediately returned to the client.
 	BlobStatus_QUEUED BlobStatus = 1
-	// ENCODED means that the blob has been encoded and is ready to be dispersed to DA Nodes
+	// ENCODED means that the blob has been Reed-Solomon encoded into chunks and is ready to be dispersed to DA Nodes.
 	BlobStatus_ENCODED BlobStatus = 2
-	// CERTIFIED means the blob has been dispersed and attested by the DA nodes
+	// CERTIFIED means the blob has been dispersed and attested by the DA nodes.
 	BlobStatus_CERTIFIED BlobStatus = 3
-	// FAILED means that the blob has failed permanently
+	// FAILED means that the blob has failed permanently.
 	BlobStatus_FAILED BlobStatus = 4
-	// INSUFFICIENT_SIGNATURES means that the blob has failed to gather sufficient attestation
+	// INSUFFICIENT_SIGNATURES means that the blob has failed to gather sufficient attestation.
```

Comment:
Is `INSUFFICIENT_SIGNATURES` also a mode of failure? I.e., is this also "permanent"?

Reply:
Yes, it is also a failure. Just a more specific failure. Anything other than `CERTIFIED` in the terminal status list is functionally the same thing. The purpose of multiple statuses is to provide context as to why the blob failed.

I've added the following style of comment to the other terminal statuses.

```
// This status is functionally equivalent to FAILED, but is used to indicate that the failure is due to an 
  // an inability to gather sufficient signatures.
```

TODO commit